packages/spotlight/Spottable/Spottable.js

```diff
@@ -393,7 +393,7 @@ const Spottable = hoc(defaultConfig, (config, Wrapped) => {
 
 		render () {
 			const {disabled, spotlightId, spotlightDisabled, ...rest} = this.props;
-			this.focusedWhenDisabled = this.isFocused && (disabled || spotlightDisabled);
+			this.focusedWhenDisabled = this.isFocused && spotlightDisabled;
```

Comment:
Maybe we could unwind this and remove `forceUpdate` that happens in blur event.

Reply:
After re-reviewing, I realized this also handles the case where a component becomes `spotlightDisabled` when focused, which is not changed with the PR.  I think it should remain.